src/main/java/com/food4thought/food4thought/service/CustomerService.java

```diff
@@ -0,0 +1,29 @@
+package com.food4thought.food4thought.service;
+
+import com.food4thought.food4thought.model.Customer;
+import com.food4thought.food4thought.repository.CustomerRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class CustomerService {
+
+    private final CustomerRepository customerRepository;
+
+    public List<Customer> getAllCustomers() {
+        return customerRepository.findAll();
+    }
+
+    public Customer getSpecificCustomer(Long id) {
+        //todo: handle optional
+        return customerRepository.findById(id).get();
```

Comment:
What you can do with optional, is basically say findById(id).orElseGet and return null inside or something that will get returned if the customer is not found. If you want you can also throw an exception with orElseThrow, depends on your usecase really.